newt/cli/project_cmds.go

```diff
@@ -200,7 +202,7 @@ func AddProjectCommands(cmd *cobra.Command) {
 	installHelpEx += "    Installs the apache-mynewt-core repository."
 	installCmd := &cobra.Command{
 		Use:     "install [repo-1] [repo-2] [...]",
-		Short:   "Install project dependencies",
+		Short:   "Install project dependencies (deprecated)",
```

Comment:
This looks fine to me, but there is also a `Deprecated` key. You could change this line to `Deprecated: "Install project dependencies"` and it would print a standard deprecation message (and still work).

Reply:
@utzig  Thanks for the tip (and the review).  That is pretty cool.  I have changed "install" and "sync" to use `Deprecated`.